README.md

```diff
@@ -1,128 +1,208 @@
+
+### Parallel test running
+When tests are run one by one, it'll cost a lot of time. `Hermione` can run tests in parallel sessions in different browsers out of the box.
+
+### Extensibility
+`WebdriverIO` provides built-in commands for browser and page manipulation. Often projects need to store some common code and reuse it through all tests. So a developer should create some helpers and include them in the tests.
+
+With `hermione` it's very simple and straightforward. You can add any number of custom commands in the hermione config and use them as `this.browser.myCustomCommand` in tests.
+
+Moreover, `hermione` provides plugins which work as a some kind of hooks. They allow a developer to prepare environement for tests and react properly to test execution events.
 
-## Конфигурация
+### Retryies for failed tests
+Integration tests use a dynamic environment with a lot of dependencies where any of them can work unstable from time to time. As a result, integration tests become red randomly and make them undetermined. It spoils all testing process.
 
-**hermione** настраивается с помощью конфигурационного файла. Путь к этому файлу можно задать с помощью опции `--conf`. Стандартное имя файла -  `.hermione.conf.js`.
+To prevent incidental fails `hermione` retryies a failed test before marking it as failed. It makes it possible to get rid of a majority of incidental fails. Number of retries can be specified for all browsers or for a separate browser.
 
-Ниже приведён пример полного конфига. Обязательными полями являются `specs` и `browsers`.
+`Hermione` reruns tests in a new browser session to exclude situation when the browser environment is a cause of this fail.
+
+### Executing a separate test
+Sometimes it is needed to run only specific tests but not all set. `Hermione` makes it possible.
+
+### Auto initilization and closing grid sessions
+All work with a grid client is incapsulated in hermione. Forget about `client.init` and `client.end` in your tests ;)
+
+## Prerequisites
+Because of `hermione` based on `WebdriverIO` you need to set up [Selenium](http://www.seleniumhq.org/) before proceed further.
+
+The simplest way to get started is to use one of the NPM selenium standalone packages like: [vvo/selenium-standalone](https://github.com/vvo/selenium-standalone). After installing it (globally) you can run your server by executing:
+```
+selenium-standalone start
+```
+
+## Quick start
+Install packages first.
+```
+npm install -g selenium-standalone
+npm install -g hermione
+```
 
+Then put `.hermione.conf.js` in the project root.
 ```javascript
-var command = require('path/to/command');
-
-module.exports = {
-    grid: 'http://localhost:4444/wd/hub',
-    baseUrl: 'http://yandex.ru/search',
-    waitTimeout: 10000,
-    debug: true,
-    sessionsPerBrowser: 2,
-    retry: 2,
-
-    plugins: {
-        'my-cool-plugin': {
-            param: 'value'
+export default {
```

Comment:
or at least with node 4.x feature support. so that an example could be used by almost everyone without extra work